androidtv/basetv/basetv.py

```diff
@@ -472,6 +472,29 @@ def _running_apps(running_apps_response):
 
         return None
 
+    @staticmethod
+    def _installed_apps(installed_apps_response):
+        """Get the installed apps output of :py:const:`androidtv.constants.CMD_INSTALLED_APPS`.
+
+        Parameters
+        ----------
+        installed_apps_response : str, None
+            The output of :py:const:`androidtv.constants.CMD_INSTALLED_APPS`
+
+        Returns
+        -------
+        list, None
+            A list of the installed apps, or ``None`` if it could not be determined
+
+        """
+
+        if installed_apps_response:
```

Comment:
Surely this is a moot point because no one would have zero installed apps, but maybe change this check to `if installed_apps_response is not None:`. This way, if the response is an empty string then you'll return an empty list. 

I'll leave it up to you.

Reply:
Valid point, will adjust! Thank you.